README.md

```diff
@@ -131,29 +131,46 @@ Deletes all terms within a specific taxonomy.
   - `taxonomy`: Taxonomy slug
 - **Returns:** Promise resolving to the deletion results
 
-#### 🔧 Additional Exports
+#### 📄 XLIFF Export Functions
 
-##### `globalSetup`
+##### `fillInXliffExportForm( page, options )`
+Fills in the XLIFF export form for bulk translation export.
+- **Parameters:**
+  - `page`: Playwright page object
+  - `options`: Configuration object
+    - `postId`: Post ID to export
+    - `postTitle`: Post title to select
+    - `languageName`: Target language name
+- **Returns:** Promise that resolves when the export form is submitted
+- **Note:** Page should be on the post list table
+
+##### `getXliffRegex( sourceLocale, targetLocale )`
+Returns a regex pattern to match XLIFF file names with the specified locales and timestamp.
+- **Parameters:**
+  - `sourceLocale`: Source language locale (e.g., 'en_US')
+  - `targetLocale`: Target language locale (e.g., 'fr_FR')
+- **Returns:** RegExp object to match XLIFF file names
+- **Note:** Converts hyphens to underscores in locales and matches timestamp pattern
```

Comment:
Isn't it the opposite ? Converts underscores to hyphens ?

Reply:
Indeed, fixed in d69aa20.